src/net.js

```diff
@@ -278,7 +278,7 @@ class Network {
        * The URL of the operation.
        * @param {string} rel
        * The type of operation. For example: `preconnect`.
-       * @param {boolean} [crossorigin]
+       * @param {boolean} [cor] crossorigin
```

Comment:
The name of the configuration property is `crossorigin`, not `core`. It does make sense to change it on line 287 as well, though it is not _technically_ required.